### PR TITLE
start on static delays

### PR DESCRIPTION
**Pull request by sbfnk**
Picks out the new features in https://github.com/epiforecasts/EpiNow2/pull/308 relating to static delays. Creating this as draft PR as it seems unfinished.

### CODE REVIEW

File: R/utilities.R [hunk 1]
```diff
@@ -311,6 +311,38 @@ convert_to_logsd <- function(mean, sd) {
   sqrt(log(1 + (sd^2 / mean^2)))
 }
 
+ discretised_lognormal_pmf <- function(meanlog, sdlog, max_d, reverse = FALSE) {
```
Comment (seabbs) [inline R/utilities.R:314]: This will cause some merge conflicts that need resolving but is unchanged.

**Review by seabbs — COMMENTED** [review]
This all looks broadly correctly but needs more testing and especially testing using the code from main. Will likely have a few merge conflicts and issues with versioning etc.

**Comment by sbfnk** [comment]
Yes, I agree, and also this may or may not be superseded by #305.

**Comment by sbfnk** [comment]
Superseded by #305.

**Pull request closed without merging**